### DISCUSSION TITLE
同一台PC 同样的设置下，shadowsocks协议可以正常使用，vmess不行

### DISCUSSION THREAD

**Discussion by GPaladinS** [2dust/v2rayNG#3853] [Q&A] [answered]
同一台PC 同样的设置下 shadowsocks协议可以正常使用，vmess不行。tcping也是显示绿色的状态。但浏览器就是打不开google。换到shadowsocks协议就可以。
所用的服务器在笔记本上和手机上都使用了很久，shadowsocks和vmess的服务器确认没问题。

**Comment by 2dust** [comment, marked as answer]
这种情况只能自己查了，主要查下系统时间和跳过证书

**Reply by GPaladinS** [reply]
感谢回复👍
检查系统时间和跳过证书是什么意思？能稍微给解释一下吗？谢谢🙏

**Reply by GPaladinS** [reply]
已经解决。检查后发现电脑上的时区设置错误。纠正后已完全正常，所有协议都可使用。再次感谢！

**Comment by Dmitry007-007** [comment]
🙏🙏🙏

**Comment by Dmitry007-007** [comment]
Дайте пожалуйста рабочий троянчик
